packages/bsky/tests/_util.ts

```diff
@@ -134,14 +135,32 @@ export const runTestServer = async (
   const bskyServer = await bsky.start()
   const bskyPort = (bskyServer.address() as AddressInfo).port
 
+  // Map pds public url to its local url when resolving from plc
+  const origResolveDid = DidResolver.prototype.resolveDid
```

Comment:
🆒 

in general, i think we should work on local testing utilities like this. Wouldn't mind for this to eventually just live on did-resolver

Reply:
That sounds interesting, I'll be the lookout for opportunities to do that.  Still sussing-out the best way to slice this.  For example, I would have loved to just stick with configuring `AtpAgent` with a custom fetch handler, which is pretty clean.  But I don't use the `AtpAgent` in the image service, since I want to stream the result and we don't have that facility yet in @atproto/api—so modifying the PLC result ended-up being the shortest path in this case.